docs/documentation/stories/moving-into-the-cli.md

```diff
@@ -0,0 +1,63 @@
+# Moving your project to Angular-CLI
```

Comment:
Overall I think we could benefit from dividing this into 2 focus areas. This is targeted at moving any app. That's great and it should do that ... but maybe we should have a walk through do it exactly for the QuickStart. Showing step by step what they do. This will give folks something to try, before doing their own. What do you think?